3.3/Dockerfile.fedora

```diff
@@ -8,7 +8,7 @@ EXPOSE 8080
 ENV NAME=ruby \
     RUBY_VERSION=3.3 \
     RUBY_SHORT_VER=33 \
-    VERSION=0
+    VERSION=
```

Comment:
Hm not sure, if there shouldn't be explicitly `VERSION=""`. I think `VERSION=` may cause different behaviour that we want. @jackorp wdyt? You know more about this issue than I do.

Reply:
That's more of a shell question. I think both VERSION="" and VERSION= in bash might end up doing the same in Ruby's ENV land. Cursory testing with interactive ruby shows they are the same. Number isn't there, that's important to us.
```console
$ VERSION= irb
irb(main):001> ENV['VERSION']
=> ""
$ VERSION='' irb
irb(main):001> ENV['VERSION']
=> ""
irb(main):002> ENV['VERSION']
```

Actually, let me re-check this with some Rails app, better be safe.

Reply:
rails-ex behaves the same with no difference whether there is `VERSION=''` or `VERSION=`. So I am ok for `VERSION=''`